src/expression.rs

```diff
@@ -124,6 +156,14 @@ fn parse_comm(pair: Pair<Rule>) -> Command {
         Rule::set_directive => Command::Set(SetDirective {
             args: pair.as_str().split(' ').skip(1).map(String::from).collect(),
         }),
+        Rule::tor_directive => {
+            let to_radix = pair.as_str().split(' ').map(String::from)
+                .collect::<Vec<String>>().pop().unwrap();
```

Comment:
You don't need to `collect` into a `Vec` and then `pop`. You can simply call `take` after the `map` - since `map` also returns an Iterator. The final `unwrap` will of-course be  necessary.

Reply:
Ok, now the line has been totally changed. I've finally found enough time to deepen the knowledge on Pest and to modify the code. I've applied `rustfmt` on `src/expression.rs` only since in the other files it was changing lines not edited by me (maybe more readable for the author in the previous format). Thanks for the suggestions.